src/Schema/Events/Click.ts

```diff
@@ -1869,3 +1869,192 @@ export interface ClickedAlertsFilters {
   changed: string
   current: string
 }
+
+/**
+ * A user clicks on Start Sendind Offers on the marketing banner in CMS
+ *
+ * This schema describes events sent to Segment from [[ClickedStartSendingOffers]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedStartSendingOffers",
+ *    context_page_owner_type: "Home",
+ *    context_page_owner_id: "",
+ *    context_page_owner_slug: "",
+ *    partner_id: "61bcda16515b038ce5000104"
+ *  }
+ * ```
+ */
+ export interface ClickedStartSendingOffers {
+  action: ActionType.clickedStartSendingOffers
+  context_page_owner_type: PageOwnerType
+  context_page_owner_id: string
+  context_page_owner_slug?: string
+  partner_id: string
+}
+
+/**
+ * A user clicks on Send Offer on the Send Offers page in CMS
+ *
+ * This schema describes events sent to Segment from [[ClickedSendOffer]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedSendOffer",
+ *    context_module: "Send offers",
+ *    context_page_owner_type: "sendOffers",
+ *    context_page_owner_id: "",
+ *    context_page_owner_slug: "",
+ *    partner_id: "61bcda16515b038ce5000104", 
+ *    arwork_id: "60de173a47476c000fd5c4cc",
+ *    price: $20,000,
+ *    last_offer_sent: 20,
+ *    new_saves: 5,
+ *    page: 1
+ *  }
+ * ```
+ */
+ export interface ClickedSendOffer {
+  action: ActionType.clickedSendOffer
+  context_page_module: string
+  context_page_owner_type: PageOwnerType
+  context_page_owner_id: string
+  context_page_owner_slug?: string
+  partner_id: string
+  artwork_id: string
+  price: number
+  last_offer_sent: number
+  new_saves: number
+  page: number
+}
+
+/**
+ * A user clicks on Add Missing Artworks Details on the Send Offers page in CMS
+ *
+ * This schema describes events sent to Segment from [[ClickedAddMissingArtworksDetails]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedAddMissingArtworksDetails",
+ *    context_module: "Send offers",
+ *    context_page_owner_type: "sendOffers",
+ *    context_page_owner_id: "",
+ *    context_page_owner_slug: "",
+ *    partner_id: "61bcda16515b038ce5000104", 
+ *    arwork_id: "60de173a47476c000fd5c4cc",
+ *    price: $20,000,
+ *    last_offer_sent: 20,
+ *    new_saves: 5,
+ *    page: 1
+ *  }
+ * ```
+ */
+ export interface ClickedAddMissingArtworksDetails {
+  action: ActionType.clickedAddMissingArtworksDetails
+  context_page_module: string
+  context_page_owner_type: PageOwnerType
+  context_page_owner_id: string
+  context_page_owner_slug?: string
+  partner_id: string
+  artwork_id: string
+  price: number
+  last_offer_sent: number
+  new_saves: number
+  page: number
+}
+
+
+/**
+ * A user clicks on pagination on the Send Offers page in CMS
```

Comment:
There are the arrows to paginate forward/backward, as well as the page numbers to jump to a specific page. Should this event be used for any type of pagination action?

Reply:
Yes! That's why I added current and next page in case the label is arrow

Reply:
Assuming we will use the `onClick` handler on `<ListPagniation>` to track this, I don't believe we will be able to distinguish if the click comes from the arrows or a page number (or which page number). The pagination component is in Palette which makes adding tracking more difficult. Would it be okay to only track the current page, and not the next page?

Reply:
Yes! I'll update here.